packages/concerto-core/lib/introspect/property.js

```diff
@@ -109,6 +109,7 @@ class Property extends Decorated {
             this.type = 'String';
             break;
         case `${MetaModelNamespace}.ObjectProperty`:
+            // should we have resolved name here?
             this.type = this.ast.type ? this.ast.type.name : null;
```

Comment:
Should we expose `resolveName` through the `decorated` instances?

Or is `resolvedName` "only" for downstream systems directly consuming and manipulating the AST?

Reply:
Is the value of this.type valid for an aliased name?